gtsfm/loader/olsson_loader.py

```diff
@@ -18,7 +18,7 @@
 
 
 class OlssonLoader(LoaderBase):
-    """Simple loader class that reads Carl Olsson's dataset from a folder on disk.
+    """Simple loader class that reads any of Carl Olsson's datasets from a folder on disk.
 
     Ref: http://www.maths.lth.se/matematiklth/personal/calle/dataset/dataset.html
```

Comment:
shouldn't this be better placed in the header docstring at the top?

Reply:
sure, the URL link you mean?